src/getTime.js

```diff
@@ -19,6 +19,9 @@
  */
 function getTime(str) {
   // write code here
+  const timeRE = /\b(2[0-3]|[01]\d):[0-5]\d\b/;
+  let time = timeRE.exec(str);
+  return (time) ? time[0] : '';
```

Comment:
parentheses are not needed.

Reply:
I'm depressed. I loved to put them so much, especially for my better understanding